code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -34,6 +34,24 @@
 		/obj/item/clothing/gloves/
 		)
 
+/obj/item/weapon/storage/belt/utility/chief
+	name = "Chief Engineer's toolbelt"
+	desc = "Holds tools, looks snazzy"
+	icon_state = "utilitybelt_ce"
+	item_state = "utility_ce"
+	..()

```

Comment:
It will. Overriding happens in procs, not definition. For a proc, the `..()` means you are calling the parent proc, after which the rest of your child proc's code will run.